final_prep/annotations.rb

```diff
@@ -16,28 +20,38 @@ def build_a_bear(name, age, fur, clothes, special_power)
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
   }
+  # shows you the details of your built bear at stored in the Hash
```

Comment:
`shows you the details` is how it kind of feels right now... but be careful with that. `returning` a value is pretty different than "showing". So I'd just recommend tightening up that explanation.